### DISCUSSION TITLE
Add low resolution (640x480)

### DISCUSSION THREAD

**Discussion by Fredobedo** [Joshua-Riek/ubuntu-rockchip#87] [General]
Dears,
Do you by chance, have any idea how I could add VGA resolution (640x480@60) for this Ubuntu ARM64 / Wayland ?

This is for making it run on a old VGA tube, a little project that I will share with you if it's working

I hoped that by just connecting it, ubuntu would drop its resolution, but not :(
Thanks

**Comment by Joshua-Riek** [comment]
I will be happy to take a look. Are you using an HDMI to VGA adapter?

**Reply by Fredobedo** [reply]
Yes I do. The goal in fact is to connect it to an arcade chassis. It's a multisync 15-24-31khz. So here the maximum resolution I may have is for instance 480p@ 60.

Thank you for helping me

**Reply by Joshua-Riek** [reply]
That is awesome! I'll have to dig up an adapter so i can check out the resolution issues, I have one somewhere,
